server/eclipse-project/src/test/java/au/org/democracydevelopers/corla/model/assertion/AssertionTests.java

```diff
@@ -660,4 +661,17 @@ public static void checkCountsDiscrepancyMap(final Assertion a, int oneOver, int
 
     assertEquals(a.cvrDiscrepancy, cvrDiscrepancy);
   }
+
+  /**
+   * Checks whether an assertion's riskMeasurement() method calculates the correct risks for
+   * varying sample numbers.
+   * @param a Assertion whose riskMeasurement() method we are testing.
+   * @param risks Map of expected sample counts to risk values.
```

Comment:
Is it the expected risk value given that sample count, assuming it has no errors? Or is there an implicit expected error rate?